src/components/portfolio/EditDialog.tsx

```diff
@@ -287,6 +287,21 @@ function reducer(state: State, action: Actions): State {
 }
 
 const useStyles = makeStyles((theme) => ({
+  dialog: {
+    '& .MuiDialog-container': {
+      minWidth: '15rem',
+    },
+    '& .MuiDialog-paper': {
+      maxHeight: '40rem',
+    },
+    '& .MuiTable-root': {
+      marginRight: '2rem',
+      minWidth: '35rem',
+    },
+  },
+  amountTableCell: {
+    paddingLeft: '6rem',
+  },
```

Comment:
The button overflow is still not fixed. The reason is not only the scrollbar. If the name column becomes too large, the width of the column that contains the input elements is too small to fit everything in a row.

![image](https://user-images.githubusercontent.com/80103802/115446071-f5b21100-a216-11eb-8fb2-ba23830ef258.png)

Setting the width of the column with the input elements to 350px ensures that there is enough space to fit everything in a row:

![image](https://user-images.githubusercontent.com/80103802/115446979-3e1dfe80-a218-11eb-8ea8-516a1c29b526.png)

Portfolio/Stock names shouldn't be that long (and if they are, they should probably be truncated), but that overflow doesn't look too bad and I'd leave it for now.

Reply:
Just setting the width works only if the text has spaces. What I did now was add an overflow-wrap rule that fixes this for all cases.